src/main/resources/templates/javaClassGraphqlType.ftl

```diff
@@ -27,9 +41,19 @@ public class ${className} <#if implements?has_content>implements <#list implemen
 </#if>
 
 <#list fields as field>
+<#if field.javaDoc?has_content>
+    /**
+     * {@link ${className}#${field.name}}
```

Comment:
I'm not sure about this. From a user's perspective, when we use the generated code, we mostly use getters and setters (we can't access the private fields), so this will add one level of indirection that may not be desirable.
I think we could simply use the doc of the field as-is on the getter and on the setter.
What do you think?

Reply:
I just didn't want to duplicate the javadoc in two places

Reply:
It doesn't really matter much in generated code I think.
Even in normal code, I would want to write a decent Javadoc on the getter and on the setter, or at the very least on the getter, because private fields are not accessible and therefore their doc.

Reply:
I think then maybe it is better then to have on the getter and do the reference in the setter:
`{@link ${className}#set${field.name?cap_first}}`

Reply:
I still don't see the benefit of this, compared to duplicate doc. When people will use the setter, we can either show them the doc directly or force them to click once more to go to the getter.

Reply:
Done as per your suggestion